prow/config/config_test.go

```diff
@@ -422,6 +422,129 @@ func TestGetGCSBrowserPrefix(t *testing.T) {
 	}
 }
 
+func TestDefaultMatches(t *testing.T) {
```

Comment:
It would help to add more confidence for this PR if you can move this test to a prior commit, to demonstrate all the cases that (expectedly) break. Then say that your change in config.go fixes the broken test cases. That way we can be confident that you're not changing behavior for all of the cases that are passing in the first and second commits.

If we already have other tests that already test the matches() function, then you can ignore my comment above.

Reply:
I will do this. There are no other focused tests, which is why I added these.